src/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Calculator from './components/Calculator';
+
+class App extends React.Component {
+  constructor(props) {
```

Comment:
[OPTIONAL] It will be better if you refactor this App.js component to be function-based too